src/main/java/nextstep/subway/application/LineService.java

```diff
@@ -15,21 +12,23 @@
 @Service
 @Transactional(readOnly = true)
 public class LineService {
-    private LineRepository lineRepository;
-    private StationRepository stationRepository;
+    private final LineRepository lineRepository;
+    private final StationRepository stationRepository;
 
     public LineService(LineRepository lineRepository, StationRepository stationRepository) {
         this.lineRepository = lineRepository;
         this.stationRepository = stationRepository;
     }
 
     @Transactional
-    public LineResponse saveLine(LineRequest lineRequest) {
-        Station upStation = stationRepository.findById(lineRequest.getUpStationId()).get();
-        Station downStation = stationRepository.findById(lineRequest.getDownStationId()).get();
-
-        Line persistLine = lineRepository.save(lineRequest.toLine(upStation, downStation));
-        return LineResponse.of(persistLine);
+    public Line saveLine(LineRequest lineRequest) {
+        Station upStation = stationRepository.findById(lineRequest.getUpStationId()).orElseThrow(RuntimeException::new);
+        Station downStation = stationRepository.findById(lineRequest.getDownStationId()).orElseThrow(RuntimeException::new);
+        Line line = lineRequest.toLine();
+        line.addLineStation(new LineStation(null, upStation, 0));
+        line.addLineStation(new LineStation(upStation, downStation, lineRequest.getDistance()));
```

Comment:
```suggestion
        Line line = lineRequest.toLine();
        line.connect(upStation, downStation);
```

로직에 영향을 주는건 아니지만 이러한 부분도 connect 등의 의미있는 메서드로 추출되면 어떨까 싶어요!! 함수이름만으로 해당 동작을 유추하게된다면 가독성이 더 높아질 것 같아요!!